docs/content/developer/advanced/custom-indexer.mdx

```diff
@@ -31,10 +32,10 @@ Data ingestion for your indexer supports several checkpoint stream sources.
 
 The most straightforward stream source is to subscribe to a remote store of checkpoint contents. IOTA provides the following buckets:
 
-- Testnet: `https://indexer.testnet.iota.cafe`
-- Devnet: `https://indexer.devnet.iota.cafe`
+- Testnet: <code>{Networks.iota_move_testnet.indexerRpc}</code>
+- Devnet: <code>{Networks.iota_move_devnet.indexerRpc}</code> 
```

Comment:
Wdyt about using CodeBlock here so people can easily copy the url?

Reply:
IMO, we should not use CodeBlocks here since I have not come across a way to render a JSX expression inside a CodeBlock, we can simply hardcode it.

Reply:
I'm not sure what you mean. This works without issues

```suggestion
- Testnet: <CodeBlock>{Networks.iota_move_testnet.indexerRpc}</CodeBlock>
- Devnet: <CodeBlock>{Networks.iota_move_devnet.indexerRpc}</CodeBlock> 
```